bundles/org.openhab.binding.evcc/README.md

```diff
@@ -52,52 +52,84 @@ Please note that some of them are only available when evcc is properly configure
+| loadpoint\<N\>#effectiveLimitTemperature | Number:Temperature | R          | Effective Temperature until which the heating device will be charged |
+| loadpoint\<N\>#vehicleTemperature        | Number:Temperature | R          | Current Temperature of the heating device                            |
 
 ### Vehicle channels
 
 Those channels exist per configured vehicle.
-Please note that you have to replace _ID_ with your vehicle id/name.
-
-| Channel                      | Type                 | Read/Write | Description                                                              |
-|------------------------------|----------------------|------------|--------------------------------------------------------------------------|
-| vehicleID#vehicleTitle       | String               | R          | Title of vehicle                                                         |
-| vehicleID#vehicleMinSoC      | Number:Dimensionless | RW         | Minimum state of charge (SoC) a vehicle should have                      |
-| vehicleID#vehicleLimitSoC    | Number:Dimensionless | RW         | Until which state of charge (SoC) should the specific vehicle be charged |
-| vehicleID#vehiclePlanEnabled | Switch               | RW         | Plan for charging enabled                                                |
-| vehicleID#vehiclePlanSoC     | Number:Dimensionless | RW         | Until which state of charge (SoC) should vehicle be charged in plan      |
-| vehicleID#vehiclePlanTime    | DateTime             | RW         | When the plan SoC should be reached                                      |
+Please note that you have to replace _\<ID\>_ with your vehicle id/name.
+
+| Channel                          | Type                 | Read/Write | Description                                                              |
+|----------------------------------|----------------------|------------|--------------------------------------------------------------------------|
+| vehicle\<ID\>#vehicleTitle       | String               | R          | Title of vehicle                                                         |
+| vehicle\<ID\>#vehicleMinSoC      | Number:Dimensionless | RW         | Minimum state of charge (SoC) a vehicle should have                      |
+| vehicle\<ID\>#vehicleLimitSoC    | Number:Dimensionless | RW         | Until which state of charge (SoC) should the specific vehicle be charged |
+| vehicle\<ID\>#vehiclePlanEnabled | Switch               | RW         | Plan for charging enabled                                                |
+| vehicle\<ID\>#vehiclePlanSoC     | Number:Dimensionless | RW         | Until which state of charge (SoC) should vehicle be charged in plan      |
+| vehicle\<ID\>#vehiclePlanTime    | DateTime             | RW         | When the plan SoC should be reached                                      |
+
+### Heating channels
```

Comment:
Sectino headers should be capitalized, please also adjust others
```suggestion
### Heating Channels
```

Reply:
done for all headlines